class CertificateNotFoundError(Exception):
    pass


class CertificateAlreadyStored(Exception):
    pass
